config/initializers/figaro.rb

```diff
@@ -1,14 +1,9 @@
 Figaro.require_keys(
-  'DASHBOARD_SECRET_KEY_BASE',
-  'IDP_SLO_URL',
-  'IDP_SP_URL',
-  'IDP_SSO_URL',
-  'LOGIN_DASHBOARD_TOKEN',
-  'NEW_RELIC_LICENSE_KEY',
-  'SMTP_ADDRESS',
-  'SMTP_DOMAIN',
-  'SMTP_PASSWORD',
-  'SMTP_USERNAME',
-  'SP_NAME',
-  'SP_PASS',
+  'saml_idp_fingerprint',
```

Comment:
nevermind, I just saw the "Coincides with cookbook update PR" part. Hooray integration!